pkg/quarterdeck/db/models/errors.go

```diff
@@ -18,9 +18,11 @@ var (
 	ErrMissingModelID      = errors.New("model does not have an ID assigned")
 	ErrMissingKeyMaterial  = errors.New("apikey model requires client id and secret")
 	ErrInvalidSecret       = errors.New("apikey secrets should be stored as argon2 derived keys")
-	ErrMissingInviteRole   = errors.New("missing role for user invite")
-	ErrInvalidInviteRole   = errors.New("invalid role for user invite")
-	ErrInvalidInviteEmail  = errors.New("invalid email for user invite")
+	ErrMissingRole         = errors.New("missing role")
+	ErrInvalidRole         = errors.New("invalid role")
+	ErrNoOwnerRole         = errors.New("organization is missing an owner")
+	ErrOwnerRoleConstraint = errors.New("organization must have at least one owner")
```

Comment:
Open to suggestions on a different name to capture this requirement

Reply:
I think the name makes sense to me.

Reply:
Me too!